src/Compilation.zig

```diff
@@ -3040,7 +3046,7 @@ fn addBuf(list: *std.ArrayList(std.posix.iovec_const), buf: []const u8) void {
 }
 
 /// This function is temporally single-threaded.
-pub fn totalErrorCount(comp: *Compilation) u32 {
+pub fn totalErrorCount(comp: *Compilation) Allocator.Error!u32 {
```

Comment:
If this function needs to allocate, then there's no point of it being separate from `getAllErrorsAlloc`. Just delete the function.